packages/@okta/vuepress-site/docs/reference/api/linked-objects/index.md

```diff
@@ -47,7 +47,7 @@ Thus, you can create chains of relationships (Jane > Bob > Joe > Frank) or termi
 
 Then, if you create another Linked Object relationship for scrum team membership, you could assign relationships to the same four users:
 
-* Bob is the scrum master for the Identity Scrum team.
+* Bob is the scrum lead for the Identity Scrum team.
 * Joe and Frank are both contributors to the team.
 
 Bob can be the `primary` for a Manager:Subordinate, an `associated` user for that same Linked Object definition, and also the `primary` for the Scrummaster:Contributor Linked Object definition.
```

Comment:
Can you change this as well: `Scrummaster:Contributor` to `Scrumlead:Contributor`?

Reply:
Good question. I'll check.

Reply:
I think "scrum lead" is fine. There was some discussion in the #inclusive-language slack channel on this topic. Some people suggested "scrum coach", but that might imply a teacher... so in the end, "scrum lead" works best.

Reply:
Also, changing the example to "Scrumlead:Contrbutor" should be ok too. It's just an example and not part of the object definition, so we should be good 👍

Reply:
Thanks @vanngo-okta :)